README.md

```diff
@@ -45,17 +44,33 @@ access_token_secret: ACCESS TOKEN SECRET
 
 [twitter-app]: https://apps.twitter.com
 
+##### Location
+
+Currently `golden-hour` is hard-coded to use Seattle's location. To use weather and sunset times for a different location, you'll need to dig into the code a bit and change some things! To fix this, check out [#11](https://github.com/alanhussey/golden-hour/issues/11)
```

Comment:
If you get the chance, can you leave notes on #11 for the lines that are hard-coded?